cockroachcloud/backups-page.md

```diff
@@ -69,19 +71,95 @@ For each table in the database, the following details display:
 
 - The number of **Rows** captured in the backup
 
-## Back up a self-hosted CockroachDB cluster and restore into a CockroachCloud cluster
+## Ways to restore data
+
+[Console Admin](console-access-management.html#console-admin) can perform the following from the Console:
+
+- [Restore a database](#restore-a-database)
+- [Restore a table](#restore-a-table)
+
+Additional ways to restore data:
+
+- [Back up a self-hosted CockroachDB cluster and restore into a CockroachCloud cluster](#back-up-a-self-hosted-cockroachdb-cluster-and-restore-into-a-cockroachcloud-cluster)
+- [Back up and restore data manually](#back-up-and-restore-data-manually)
+
+### Restore a database
+
+To restore a database:
+
+1. Find the cluster backup containing the database you want to restore, and click the number in the corresponding **Databases** column.
+
+    The **Databases** view displays.
```

Comment:
Took me a sec to figure out what this is saying.  I guess in my mind I might phrase it as "The Databases view is displayed" - but perhaps that is a passive voice that we are avoiding?

(Aside: makes me think of "A wild **databases** view appears ..."